src/transport_performance/utils/defence.py

```diff
@@ -120,7 +126,12 @@ def _is_expected_filetype(pth, param_nm, check_existing=True, exp_ext=".zip"):
     _, ext = os.path.splitext(pth)
     if check_existing and not os.path.exists(pth):
         raise FileExistsError(f"{pth} not found on file.")
```

Comment:
This should be a `FileNotFoundError` rather than a `FileExistsError`. The latter indicates that the file is present, when it has been determined that it is not.

Reply:
Ah, my bad. Will patch now.